string[] tonode
---
float64[] totalTime
